DigitalLearningSolutions.Web.Tests/Controllers/LearningPortal/LearningPortalControllerTests.cs

```diff
@@ -74,9 +75,9 @@ public void SetUp()
                 candidateAssessmentDownloadFileService,
                 searchSortFilterPaginateService
             );
-            controller.ControllerContext = new ControllerContext { HttpContext = new DefaultHttpContext { User = user } };
+            controller.ControllerContext = new ControllerContext
+                { HttpContext = new DefaultHttpContext { User = user } };
             controller = controller.WithMockTempData();
-
         }
     }
 }

```

Comment:
It looks like there's no tests at all for the method I made my small change to. Since HEE wrote the method I'm not sure if we want to spend the time writing tests for it. @SteveJacksonSoft do you have an opinion?